apps/browser/qml/pages/LoginsPage.qml

```diff
@@ -33,6 +33,11 @@ Page {
         return hostname
     }
 
+    function copyPassword(password) {
+        Clipboard.text = password
```

Comment:
Not completely sure if we should have an extra clipboard property for sensitive data so that vkb would not show. That said, that would be a new task not related to this one per se.

Reply:
It's a really good idea in my opinion.

Reply:
I looked into this further. Unfortunately there's no current capability for flagging sensitive data in the clipboard. It wouldn't be hard to enforce in the keyboard, the implementation difficulty is in how to lett the keyboard know. QClipboard doesn't have any feature to flag sensitive content. A `sensitive` flag could be sent directly to the keyboard rather than via the clipboard when the sensitive contents is set, it just requires some work in choosing the best way to communicate it to the keyboard.

This is just background info. I'll resolve this conversation for now and maybe we can return to it in the future.